src/components/FeaturedProject.tsx

```diff
@@ -33,7 +33,7 @@ function FeaturedProject(props: featuredProjectProps) {
                     <iframe
                         width="100%"
                         height="100%"
-                        src={`https://www.youtube.com/embed/${props.videoEmbedID}`}
+                        src={`https://www.youtube.com/embed/${props.videoEmbedID}&autoplay=1&mute=1`}
```

Comment:
Can we also disable hover?

Reply:
I think you just need `style="pointer-events: none`

Reply:
If we disable hover how can they pause/unpause and unmute?

Reply:
These are just meant to be short video snippets that encourage people to visit the project urls themselves to learn more and interact with the deployed apps.